arraycontext/impl/pytato/fake_numpy.py

```diff
@@ -62,7 +63,7 @@ def _get_fake_numpy_linalg_namespace(self):
     def __getattr__(self, name):
         if name in self._pt_funcs:
             from functools import partial
-            return partial(rec_map_array_container, getattr(pt, name))
```

Comment:
`multimap` is much more expensive than `map`. Why not separate unary and $n$-ary functions to avoid this expense when it is in fact avoiable?

Reply:
Yep, fair. Will do.

Reply:
See [d9d3fff](https://github.com/inducer/arraycontext/pull/219/commits/d9d3fff6608d2cbe4f8e6e1bed97b5677be1d565).